lib/instrument/transformer/nskeyed.js

```diff
@@ -203,11 +205,87 @@ class XCTCapabilities extends BaseArchiveHandler {
 }
 
 class NSNull extends BaseArchiveHandler {
+  static isNSObject = true;
   decodeArchive () {
     return null;
   }
 }
 
+class XCTestConfiguration extends BaseArchiveHandler {
+  static _default = {
+    aggregateStatisticsBeforeCrash: {
+      XCSuiteRecordsKey: {}
+    },
+    automationFrameworkPath: '/Developer/Library/PrivateFrameworks/XCTAutomationSupport.framework',
```

Comment:
Probably this works only for Xcode 13.2+(?) since the framework was added since 13.2. (probably Xcode 14+ should be safe since the framework had no much file then). Perhaps documentation should be sufficient

Reply:
This fucntion is to simulate xcode behaviour, so it won't interrupt normal function

Reply:
it would be sense to keep this as a comment in the code